src/sass/modules/table-list.scss

```diff
@@ -35,3 +35,17 @@
   font-size: 12px;
   padding: $pad-small;
 }
+
+.table-empty-list__body {
```

Comment:
I think this can probably be done with a single class

Reply:
were you thinking that I should move the class up out of `table-list` and the patient one as well, and just have some 'generic' `.empty-list` class both can use?

Reply:
I think where they're defined is fine.  I just think it can be done with one class rather than two